reactor-core-2/src/test/java/com/baeldung/reactor/flux/parallelflux/ParallelFluxUnitTest.java

```diff
@@ -0,0 +1,56 @@
+package com.baeldung.reactor.flux.parallelflux;
+
+import com.baeldung.reactor.fluxvsparallelflux.Fibonacci;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.Test;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.ParallelFlux;
+import reactor.core.scheduler.Schedulers;
+import reactor.test.StepVerifier;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+@Slf4j
+public class ParallelFluxUnitTest {
```

Comment:
What is different about the two tests here, what are you trying to demonstrate, the same thing in each test or is there a different concept being communicated? I didn't understand the distinction